tests/tests_integration/test_api/test_sequences.py

```diff
@@ -48,8 +48,9 @@ def root_asset(cognite_client: CogniteClient) -> Asset:
         name="integration_test:root_asset",
         external_id="integration_test:root_asset",
     )
-    retrieved = cognite_client.assets.retrieve_multiple(external_ids=[root_asset.external_id], ignore_unknown_ids=True)
-    if retrieved:
+    if retrieved := cognite_client.assets.retrieve_multiple(
+        external_ids=[root_asset.external_id], ignore_unknown_ids=True
+    ):
```

Comment:
This change requires an extra line compared to not using the walrus 🤔

Reply:
I find this one more readable than the original. 🤷 

<img width="626" alt="Screenshot 2023-10-11 at 10 26 28" src="https://github.com/cognitedata/cognite-sdk-python/assets/60771923/59ca05a0-1a69-495b-bb3c-57f954d54137">

Figure 1: After with walrus.

<img width="779" alt="Screenshot 2023-10-11 at 10 27 44" src="https://github.com/cognitedata/cognite-sdk-python/assets/60771923/b3339a6a-5979-4fac-b532-807a8d46a9fd">

Figure 2: Before with line stretching out further.

🤷‍♂️

Reply:
Up to you 😉

Reply:
Alright, I'll make an executive decision here. 😛